tests/nullish_coalescing/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -28,7 +28,7 @@ foo ? bar ?? foo : baz;
 
 foo ?? (bar ?? baz);
 
-foo ?? baz || baz;
+(foo ?? baz) || baz;
```

Comment:
when you start using a fixed parser, the `foo ?? bar || baz` above will start to error, so might want to remove that case?

can you add a test for `(foo ?? bar) || baz` to make sure it doesn't strip the parens anymore?

Reply:
Good points